### ISSUE TITLE
Migration Issue - There was an error while loading site data. - Unexpected Error

### ISSUE THREAD

**Issue by knmurphy** [sveltia/sveltia-cms#142]
Hi! 👋🏻 😄  Thank you to all who contribute to making this awesome project! Once I get more familiar with it I hope I can contribute more directly to its success. 

At the moment, I'm stuck migrating to Sveltia CMS from DecapCMS. I was able to migrate from git-gateway to github oauth but after I authorize and get redirected I ended up with this error:

> There was an error while loading site data
> Unexpected Error
> 

<img width="950" alt="Screen Shot 2024-05-13 at 2 54 08 PM" src="https://github.com/sveltia/sveltia-cms/assets/2067836/1b9aad5b-93dd-4f7a-8499-4e379a15ad73">

My developer console shows this error:

> Cannot read properties of null (reading sveltia-cms.js:266
> 'ref') undefined
> Fm @ sveltia-cms.js:266

<img width="506" alt="Screen Shot 2024-05-13 at 2 59 56 PM" src="https://github.com/sveltia/sveltia-cms/assets/2067836/bd2872c9-8ce1-42d0-a372-66b17894ec0f">

Here is the ```config.yml``` block for auth

```
# The backend option specifies how to access the content for your site
backend:
  name: github
  repo: <MY/REPO> # My repo path, removed for this post
  # optional, defaults to master
  branch: master # The branch to update 
```
I looked through the issues and discussions but couldn't work around it on my own. Any idea what might solve it? Thanks!

**Comment by kyoshino** [comment]
@knmurphy Thanks for reporting the issue! I’m checking it now.

**Comment by kyoshino** [comment]
Found an issue your config file:

https://github.com/knmurphy/intuicion-cafe/blob/2cc9def31df334c4715a49e29a419fbcfc732006/admin/config.yml#L6

`knmurphy/intuicion.cafe` has to be `knmurphy/intuicion-cafe`. (Replace a dot with a hyphen)

I’ll fix my code anyway, but you’ll still get an error until the repo name is fixed 😉

**Comment by knmurphy** [comment]
Wow, thank you for this! I am trying the fix now 👍🏻

**Comment by kyoshino** [comment]
Just shipped [v0.27.1](https://github.com/sveltia/sveltia-cms/releases/tag/v0.27.1) to fix an error on my side!

**Comment by knmurphy** [comment]
You are correct, that fixed it! Thanks so much. 

I am really excited to dig into this, I have been so frustrated with netlifycms in the past so I'm excited to find an active replacement 🙌🏻
